scene/gui/control.h

```diff
@@ -149,6 +149,12 @@ class Control : public CanvasItem {
 		TEXT_DIRECTION_INHERITED = TextServer::DIRECTION_INHERITED,
 	};
 
+	enum AutoTranslateMode {
+		AUTO_TRANSLATE_MODE_INHERIT, // Same as parent `Control`/`Window`.
+		AUTO_TRANSLATE_MODE_ALWAYS, // Always auto translate.
+		//AUTO_TRANSLATE_MODE_DISABLED, // Never auto translate. Commented until compatibility breakage is allowed, where it will replace the `auto_translate` property.
```

Comment:
Along with the suggestions to outright replace `auto_translate`, it would make more sense to shift **AUTO_TRANSLATE_MODE_DISABLED** as the first enum constant (0), in my opinion.

Reply:
There doesn't seem to be an actual pattern with this sort of stuff in the codebase, even in the file itself. I just followed how `Node` does it with `process_mode` (where `DISABLED` is last).